orchagent/mplsrouteorch.cpp

```diff
@@ -142,74 +146,97 @@ void RouteOrch::doLabelTask(Consumer& consumer)
                     if (fvField(i) == "mpls_pop")
                         pop_count = to_uint<uint8_t>(fvValue(i));
 
-                    if (fvField(i) == "blackhole")
-                        blackhole = fvValue(i) == "true";
+                    if (fvField(i) == "nexthop_group")
+                        nhg_index = fvValue(i);
                 }
-                vector<string> ipv = tokenize(ips, ',');
-                vector<string> alsv = tokenize(aliases, ',');
-                vector<string> mpls_nhv = tokenize(mpls_nhs, ',');
 
-                /* Resize the ip vector to match ifname vector
-                 * as tokenize(",", ',') will miss the last empty segment. */
-                if (alsv.size() == 0 && !blackhole)
+                SWSS_LOG_INFO("Label route %u has nexthop_group: %s, ips: %s, "
```

Comment:
same here, This can fill up the logs if you plan to support labels at scale.

Reply:
Removed